ui/widgets/nuxeo-operation-button.js

```diff
@@ -54,13 +54,17 @@ import '../actions/nuxeo-action-button-styles.js';
           sync-indexing$="[[syncIndexing]]"
           async$="[[async]]"
           poll-interval="[[pollInterval]]"
+          on-poll-start="_onPollStart"
+          on-poll-update="_onPollUpdate"
+          on-poll-error="_onPollError"
+          on-poll-aborted="_onPollAbort"
         >
         </nuxeo-operation>
```

Comment:
Since we are using the same instance of nuxeo-operation, is it possible that if the user clicks twice on this button (even on different folders) and the polling times coincide, that they will abort each other? If so, we might consider using the `uncancelable` attribute.

Reply:
You might be right 🤔 I'll check this, thanks.